foobar.txt

```diff
@@ -0,0 +1 @@
+xyz
```

Comment:
I think `abc` would have been preferrable, but let's keep it like this now that it's in there